src/components/NewsWidget/NewsWidget.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useContext, useRef } from 'react';
+import Scrollbars from 'react-custom-scrollbars';
+import { ArrowLeft, ArrowRight } from 'react-feather';
+import ReactMarkdown from 'react-markdown';
+import Slider, { Settings } from 'react-slick';
+import remarkGfm from 'remark-gfm';
+import { ThemeContext } from 'styled-components';
+import Earth from 'src/assets/images/earth.png';
```

Comment:
I think this image needs to be added into PR

Reply:
done